--- src/Interfaces/scilab.c	2017-10-09 10:44:05.000000000 +0200
+++ src/Interfaces/scilab.c	2024-11-04 16:26:22.642577801 +0100
@@ -33,7 +33,7 @@
 // old chroot:  char ans_str[]="\n-->";
 char ans_str[]="\n";
 
-char *nameofcmd="scilab -nw -ns";
+char *nameofcmd="scilab-cli -nw -ns";
 int precision=9;
 char header[]="\
 ";
@@ -58,7 +58,7 @@
 
 /***************** Nothing should need change hereafter *****************/
 
-char *progname="scilab";
+char *progname="scilab-cli";
 
 /* check for security violations in command string */
 void check_parm(char *p)
